packages/effect/src/BigInt.ts

```diff
@@ -558,3 +558,68 @@ export const toNumber = (b: bigint): Option.Option<number> => {
   }
   return Option.some(Number(b))
 }
+
+/**
+ * Takes a string and returns an `Option` of `bigint`.
+ *
+ * If the string is empty or contains characters that cannot be converted into a `bigint`,
+ * it returns `Option.none()`, otherwise, it returns `Option.some(bigint)`.
+ *
+ * @param s - The string to be converted to a `bigint`.
+ * @returns An `Option` type that is either `some(bigint)` if the string can be converted,
+ *          or `none()` if it cannot.
```

Comment:
```suggestion
 * @param s - The string to be converted to a `bigint`.
```

Redundant (repetition of function description)